doc/performance-investigations/0012-increase-machine-type-core-count/summary.md

```diff
@@ -0,0 +1,49 @@
+# Investigate Machine Type Architecture Performance
+
+EQ Runner currently uses a machine type of `custom-4-4096` (4 vCPU cores, 4096MB of RAM). This investigation looked at the impact of machine type on the performance of the runner application, focusing on the number of CPU cores as Runner is primarily a CPU bound application with limited network IO to Datastore and Redis.
+
+The investigation aims to determine whether latencies as measured by the Locsut client improve when using larger machines but utilizing roughly the same percentage of CPU.
+
+## Benchmark profile
+
+| Option                 | Value                        |
+| ---------------------- | ---------------------------- |
+| Requests file          | census_household_gb_eng.json |
+| Run time               | 20m                          |
+| User wait time minimum | 1                            |
+| User wait time maximum | 2                            |
+| Clients                | 256                          |
+| Number of workers      | 7                            |
+| Number of threads      | 7                            |
+
+This test replicates the current settings for the daily test. Note that the baseline test ran with 64 clients as per the daily test.
```

Comment:
What baseline test?

Reply:
I can remove this from here, as it references the comparison against 4 cores and 1 pod in the c type test.